omnibus/config/software/chefdk.rb

```diff
@@ -39,17 +39,13 @@
 
 relative_path "chef-dk"
 
-if windows?
-  dependency "ruby-windows"
-  dependency "ruby-windows-devkit"
-else
-  dependency "libffi" if debian?
-  dependency "ruby"
-end
+dependency "ruby"
 
 dependency "rubygems"
 dependency "bundler"
 dependency "appbundler"
+# windows does not have native readline support with compiled ruby
+dependency "rb-readline" if windows?
```

Comment:
Minor nit: I would put this in the project definition instead of here because chefdk (the software) shouldn't technically care about the exactly it gets ruby or even if it gets readline.  chefdk the project on the other hand probably cares about the platforms its packaging for.

This way, if we every get to refactoring the ruby software definition, we can move all the ruby related pre and post install steps into one place.